src/SFA.DAS.ApprenticeCommitments/Apis/ApprenticeLoginApi/SendInvitationRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Newtonsoft.Json;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.ApprenticeCommitments.Apis.ApprenticeLoginApi
+{
+    public class SendInvitationRequest : IPostApiRequest
+    {
+        public string PostUrl => $"/invitations/{ClientId}";
```

Comment:
Maybe have the `ClientId` as ctor param - similar to what is done for the `GetApprenticeshipDetailsRequest`